css-canvas-medical/index.html

```diff
@@ -0,0 +1,125 @@
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body class="font-avenir font-body">
+  <div class="container">
+    <div class="row font-gray header-height">
+      <div class="column-half">
+        <img class="canvas-logo" src="images/logo-blue.svg" alt="Canvas Logo">
+      </div>
+      <div class="column-one-tenth">
+        <a class="header-button product" href="">Product</a>
+      </div>
+      <div class="column-one-tenth">
+        <a class="header-button implementation" href="">Implementation</a>
+      </div>
+      <div class="column-one-tenth">
+        <a class="header-button about-us" href="">About Us</a>
+      </div>
+      <div class="column-one-tenth">
+        <a class="header-button blog" href="">Blog</a>
+      </div>
+      <div class="column-one-tenth">
+        <a class="header-button contact-us" href="">Contact Us</a>
+      </div>
+    </div>
+    <div class="row justify-center font-white">
+      <div class="column-ninety background-hero">
+        <div class="hero-text">
+          <h1 class="font-tiempos-headline">The EMR Reimagined for <br>Primary Care</h1>
+          <p>
+            Can't remember the last time you talked with a patient without focusing <br>
+            on the EMR? Canvas lets you rediscover joy in the work you do. Capture <br>
+            meaningful data, identify gaps in care, and move seamlessly between <br>
+            tasks within the platform. You stay present with patients while your <br>
+            whole care team stays up-to-speed. Be a better doctor with Canvas, <br>
+            designed for primary care providers with the help of primary care <br>
+            providers.
+          </p>
+          <div>
+            <a class="get-started-button" href="">Get Started</a>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="row background-clear font-black-coral justify-center">
+      <div class="column align-text-center clinicians-header">
+        <h2 class="font-tiempos-headline">Clinicians Prefer Canvas</h2>
+        <p>
+          Physician-owned primary care practices across the country have <br>
+          already made Canvas the fabric of their care.
+        </p>
+      </div>
+    </div>
+    <div class="row wrap background-clear font-black-coral clinicians-list justify-center">
+      <div class="column-44-4">
+        <h3>Cascade Family Practice, Portland OR</h3>
+      </div>
+      <div class="column-44-4">
+        <h3>Ward Medical Clinic, Bonifay FL</h3>
+      </div>
+      <div class="column-44-4">
+        <h3>Temescal Creek Medicine, Oakland CA</h3>
+      </div>
+      <div class="column-44-4">
+        <h3>Caribou Medical, Towanda PA</h3>
+      </div>
+      <div class="column-44-4">
+        <h3>San Juan Healthcare, Friday Harbor WA</h3>
+      </div>
+      <div class="column-44-4">
+        <h3>Bay Tree Family Care, Red Bay AL</h3>
+      </div>
```

Comment:
Instead of this `44.4%` precise measurement, I would use `column-half` and just apply some padding.

Reply:
Yeah, on the next challenge, the four-column, I started doing it like that, xD